lib/travis/build/bash/travis_cmd.bash

```diff
@@ -54,9 +54,9 @@ travis_cmd() {
     fi
     result="${?}"
     if [[ -n "${secure}" && "${result}" -ne 0 ]]; then
-      echo -e "${ANSI_RED}The previous command failed, possibly due to a malformed secure environment variable.${ANSI_CLEAR}
-${ANSI_RED}Please be sure to escape special characters such as ' ' and '$'.${ANSI_CLEAR}
-${ANSI_RED}For more information, see https://docs.travis-ci.com/user/encryption-keys.${ANSI_CLEAR}"
+      echo -e "${ANSI_RED}We were unable to parse one of your secure environment variables.${ANSI_CLEAR}
+${ANSI_RED}Please make sure to escape special characters such as ' ' and $ with .${ANSI_CLEAR}
+${ANSI_RED}For example, thi$isanexample would be typed as thi$isanexample. See https://docs.travis-ci.com/user/encryption-keys.${ANSI_CLEAR}"
```

Comment:
I am not sure if I understand this sentence.

Reply:
Which sentence?

Reply:
> For example, thi$isanexample would be typed as thi$isanexample.

`thi$isanexample` appears exactly the same, and does not seem like a good example.

Reply:
Oh, again the formatting in markdown escaping haha

The second one should be `thi\$isanexample`